packages/common/src/store/pages/profile/reducer.ts

```diff
@@ -227,6 +229,12 @@ const actionsMap = {
   },
   [FETCH_COLLECTIONS_FAILED](state, action) {
     return updateProfile(state, action, { collectionStatus: Status.ERROR })
+  },
+  [REFRESH_WALLET_COLLECTIBLES](state, action) {
```

Comment:
nit: do we even need these here

Reply:
dont think we do. mechanically added and forgot to remove. good eyes